lms/djangoapps/certificates/api.py

```diff
@@ -344,31 +343,30 @@ def cert_generation_enabled(course_key):
 
 
 def generate_example_certificates(course_key):
-    """Generate example certificates for a course.
+    """Generate example (PDF) certificates for a course.
 
-    Example certificates are used to validate that certificates
-    are configured correctly for the course.  Staff members can
-    view the example certificates before enabling
-    the self-generated certificates button for students.
+    Example certificates were used to validate that certificates were configured correctly for the course.  Staff
+    members could view the example certificates before enabling the self-generated certificates button for students.
 
-    Several example certificates may be generated for a course.
-    For example, if a course offers both verified and honor certificates,
-    examples of both types of certificate will be generated.
+    [07/20/2021 Update]
+    This function was updated to remove the references to queue.py, which has been removed as part of MICROBA-1227, and
+    no longer can fulfill the function it was originally created for. There is further cleanup around PDF certificate
+    generation code, part of DEPR-155, that will remove this function. See DEPR-155 and MICROBA-1094 for additional
+    info.
 
-    If an error occurs while starting the certificate generation
-    job, the errors will be recorded in the database and
-    can be retrieved using `example_certificate_status()`.
+    It may be important to note that this functionality has been broken since 2018 when the ability to generate PDF
+    certificates was ripped out of edx-platform. This will be removed as part of MICROBA-1394.
 
     Arguments:
         course_key (CourseKey): The course identifier.
 
     Returns:
         None
-
     """
-    xqueue = XQueueCertInterface()
-    for cert in ExampleCertificateSet.create_example_set(course_key):
-        xqueue.add_example_cert(cert)
+    log.warning(
+        "Generating example certificates is no longer supported. Skipping generation of example certificates for "
+        f"course {course_key}"
+    )
```

Comment:
This function will be cleared up later and I thought a simple log message will suffice until we can start the removal of the PDF generation code.

AFAIK, the `generate example certificates` button only appears on the Instructor Dashboard and _only_ for courses that are configured for PDF certificates. The (deprecated) course setting `Certificate Web/HTML View Enabled` in Studio must be set to **false** for this button to appear. This setting defaults to `true` for new courses.

I don't think anyone will notice this change.